app/src/main/java/salesian/university/broker/TopicManager.java

```diff
@@ -0,0 +1,27 @@
+package salesian.university.broker;
+
+import java.util.*;
+
+public class TopicManager {
+    private final Map<String, List<String>> subscriptions;
+
+    public TopicManager() {
+        this.subscriptions = new HashMap<>();
+    }
+
+    public void createTopic(String topic) {
+        subscriptions.putIfAbsent(topic.toLowerCase(), new ArrayList<>());
+    }
+
+    public void addSubscriber(String topic, String consumerUrl) {
+        String topicLowerCase = topic.toLowerCase();
```

Comment:
Dont' you think we can have a utils class for converting topic into lower case? I see you've implemented in different lines.